### PR TITLE
A PR for migrating Cineast features

### PR DESCRIPTION
**Pull request by ppanopticon**
This PR contains Cineast features that we have decided to migrate.

See #94 for discussion

### CODE REVIEW

File: vitrivr-engine-module-features/src/main/kotlin/org/vitrivr/engine/module/features/feature/ehd/EHDRetriever.kt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * @see [EHD]
+ *
+ * @author Ralph Gasser
+ * @version 1.0.0
+ */
+class EHDRetriever(field: Schema.Field<ImageContent, FloatVectorDescriptor>, query: ProximityQuery<Value.FloatVector>, context: QueryContext) : AbstractRetriever<ImageContent, FloatVectorDescriptor>(field, query, context) {
+
+    companion object {
+        /** [LinearCorrespondence] for [EHDRetriever]. Maximum distance is taken from Cineast implementation. */
+        private val CORRESPONDENCE = LinearCorrespondence(16 / 4f)
```
Comment (lucaro) [inline vitrivr-engine-module-features/src/main/kotlin/org/vitrivr/engine/module/features/feature/ehd/EHDRetriever.kt:28]: Soo, `4f`? 😜

Comment (ppanopticon) [reply vitrivr-engine-module-features/src/main/kotlin/org/vitrivr/engine/module/features/feature/ehd/EHDRetriever.kt:28]: :-)

As written: It's been taken from the Cineast implementation :P

Comment (lucaro) [reply vitrivr-engine-module-features/src/main/kotlin/org/vitrivr/engine/module/features/feature/ehd/EHDRetriever.kt:28]: yeah, but I don't think we need to necessarily backport sins of the past

Comment (ppanopticon) [reply vitrivr-engine-module-features/src/main/kotlin/org/vitrivr/engine/module/features/feature/ehd/EHDRetriever.kt:28]: I agree. 

Frankly, I was assuming there to be some "educational" value in the notation. Sometimes I express thing like this myself to hint at how the value was derived.

**Review by lucaro — APPROVED** [review]

**Pull request merged**